testsuite/forge-cli/src/main.rs

```diff
@@ -465,8 +474,28 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
             .with_success_criteria(SuccessCriteria::new(
                 4000,
                 10000,
+                true,
                 Some(Duration::from_secs(240)),
             )),
+        "consensus_stress_test" => config
+            .with_network_tests(&[&ContinuousProgressTest { target_tps: 100 }])
+            .with_initial_validator_count(NonZeroUsize::new(10).unwrap())
+            .with_genesis_helm_config_fn(Arc::new(|helm_values| {
+                helm_values["chain"]["epoch_duration_secs"] = 60.into();
+            }))
+            .with_node_helm_config_fn(Arc::new(|helm_values| {
+                helm_values["consensus"]["max_block_txns"] = 50.into();
+                helm_values["consensus"]["round_initial_timeout_ms"] = 500.into();
+                helm_values["consensus"]["round_timeout_backoff_exponent_base"] = 1.0.into();
+                helm_values["consensus"]["quorum_store_poll_count"] = 1.into();
+            }))
+            .with_emit_job(EmitJobRequest::default().mode(EmitJobMode::ConstTps { tps: 100 }))
```

Comment:
Any specific reason we want to send such low TPS?

Reply:
I'll add this as a comment to the code, I put it in the description of the PR:

"Small constant traffic, small blocks and fast rounds, and short epochs.

maximizing number of rounds and epochs within a given time"